mne/tests/test_epochs.py

```diff
@@ -412,6 +412,20 @@ def test_reject_epochs():
     n_events = len(epochs.events)
     data = epochs.get_data()
     n_clean_epochs = len(data)
+
+    # apply rejection through constructor and drop_bad_epochs() method and
+    # check if they match
+    epochs_noreject = Epochs(raw, events1, event_id, tmin, tmax, picks=picks,
+                             baseline=(None, 0))
+    epochs_noreject.drop_bad_epochs(reject=reject, flat=flat)
+    assert_array_equal(epochs_noreject.get_data(), epochs.get_data())
+
+    # do apply_reject() a second time with more conservative thresholds
+    reject_crazy = dict(grad=1000e-15, mag=4e-15, eeg=80e-9, eog=150e-9)
+    epochs.drop_bad_epochs(reject=reject_crazy, flat=flat)
+    epochs_noreject.drop_bad_epochs(reject=reject_crazy, flat=flat)
```

Comment:
so it means that if you call twice drop_bad_epochs then you call twice _get_data_from_disk and read twice from disk?